src/olympia/constants/base.py

```diff
@@ -359,6 +359,9 @@
 # First version we consider as "Fenix".
 MIN_VERSION_FENIX = '79.0a1'
 
+# Last version we consider as "Fennec"
+MAX_VERSION_FENNEC = '68.*'
```

Comment:
If we consider the minimum version of Fenix to be 79, and the maximum version of Fennec to be 68, what do we consider versions from 69 to 78 to be?

Reply:
One thing I could do for consistency is remove `MIN_VERSION_FENIX` completely, and use `MAX_VERSION_FENNEC` to find the beginning of the forbidden range instead. The reason I didn't is because `68.*` is an invalid min version (since it has a `*`) so I couldn't do a simple `.get()` to find the beginning of the forbidden range. But I could work around that and pick the first version after... (it would likely be `69.0`, even though that version doesn't really exist for Android)